src/index.html

```diff
@@ -17,11 +17,317 @@
+  Price:
+  <span class="card__price-value">$2,199</span>
+</p>
+<a
+  href="#"
+  class="card__buy"
+  data-qa="hover"
+>
+  BUY
+</a>
+</article>
+
+<article
+class="card"
+data-qa="card"
+>
+<img
+  src="./images/imac.jpeg"
+  alt="Product Image"
+  class="card__image"
+/>
+<h2 class="card__title">
+  APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+</h2>
+<p class="card__code">Product code: 195434</p>
+
+<div class="stars stars--4">
+  <span class="stars__star"></span>
+  <span class="stars__star"></span>
+  <span class="stars__star"></span>
+  <span class="stars__star"></span>
+  <span class="stars__star"></span>
+</div>
+
+<p class="card__reviews">Reviews: 5</p>
+<p class="card__price">
+  Price:
+  <span class="card__price-value">$2,199</span>
+</p>
+<a
+  href="#"
+  class="card__buy"
+  data-qa="hover"
+>
+  BUY
+</a>
+</article>
+
+<article
+class="card"
+data-qa="card"
+>
+<img
+  src="./images/imac.jpeg"
+  alt="Product Image"
+  class="card__image"
+/>
+<h2 class="card__title">
+  APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+</h2>
+<p class="card__code">Product code: 195434</p>
+
+<div class="stars stars--4">
+  <span class="stars__star"></span>
+  <span class="stars__star"></span>
+  <span class="stars__star"></span>
+  <span class="stars__star"></span>
+  <span class="stars__star"></span>
+</div>
+
+<p class="card__reviews">Reviews: 5</p>
+<p class="card__price">
+  Price:
+  <span class="card__price-value">$2,199</span>
+</p>
+<a
+  href="#"
+  class="card__buy"
+  data-qa="hover"
+>
+  BUY
+</a>
+</article>
+
+<article
+class="card"
+data-qa="card"
+>
+<img
+  src="./images/imac.jpeg"
+  alt="Product Image"
+  class="card__image"
+/>
+<h2 class="card__title">
+  APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+</h2>
+<p class="card__code">Product code: 195434</p>
+
+<div class="stars stars--4">
+  <span class="stars__star"></span>
+  <span class="stars__star"></span>
+  <span class="stars__star"></span>
+  <span class="stars__star"></span>
+  <span class="stars__star"></span>
+</div>
+
+<p class="card__reviews">Reviews: 5</p>
+<p class="card__price">
+  Price:
+  <span class="card__price-value">$2,199</span>
+</p>
+<a
+  href="#"
+  class="card__buy"
+  data-qa="hover"
+>
+  BUY
+</a>
+</article>
+
+<article
+class="card"
+data-qa="card"
+>
+<img
+  src="./images/imac.jpeg"
+  alt="Product Image"
+  class="card__image"
+/>
+<h2 class="card__title">
+  APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+</h2>
+<p class="card__code">Product code: 195434</p>
+
+<div class="stars stars--4">
+  <span class="stars__star"></span>
+  <span class="stars__star"></span>
+  <span class="stars__star"></span>
+  <span class="stars__star"></span>
+  <span class="stars__star"></span>
+</div>
+
+<p class="card__reviews">Reviews: 5</p>
+<p class="card__price">
+  Price:
+  <span class="card__price-value">$2,199</span>
+</p>
+<a
+  href="#"
+  class="card__buy"
+  data-qa="hover"
+>
+  BUY
+</a>
+</article>
```

Comment:
Each product card is identical, which suggests a misunderstanding of the task requirements. Each card should represent a different product with unique details such as image, title, product code, reviews, and price.